cpp/src/arrow/flight/integration_tests/test_integration.h

```diff
@@ -28,22 +28,27 @@
 
 namespace arrow {
```

Comment:
This isn't your mistake, but this file is missing a `#pragma once` up top - hence the CI failure.